src/main/java/uk/gov/mca/beacons/api/gateways/NoteGatewayImpl.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.mca.beacons.api.gateways;
+
+import uk.gov.mca.beacons.api.domain.Note;
+import uk.gov.mca.beacons.api.jpa.NoteJpaRepository;
+import uk.gov.mca.beacons.api.jpa.entities.NoteEntity;
+import uk.gov.mca.beacons.api.mappers.NoteMapper;
+
+public class NoteGatewayImpl implements NoteGateway {
+
+  private NoteJpaRepository noteJpaRepository;
+
+  public NoteGatewayImpl(NoteJpaRepository noteRepository) {
+    this.noteJpaRepository = noteRepository;
+  }
+
+  @Override
+  public void save(Note note) {
```

Comment:
should our gateways (generally) expose create(...) and update(...) rather than save?

Reply:
or is that a service-level concern?

Reply:
Umm good question, I basically went along with what the other gateways had... being consistent? 😬 

But maybe we need to reconsider the terms we've been using - there's not many so might be a good time to change!

[account holder](https://github.com/mcagov/beacons-service/blob/main/src/main/java/uk/gov/mca/beacons/api/gateways/AccountHolderGateway.java)
[owner](https://github.com/mcagov/beacons-service/blob/main/src/main/java/uk/gov/mca/beacons/api/gateways/OwnerGateway.java)

Reply:
the current `AccountHolder save(...)` is (as far as i can tell) an **insert** but my current ticket is for account holder updates, so a **save** which inserts doesn't make sense for me 🤷🏼‍♂️

Reply:
(to recap what is in Slack) yeah, that makes sense! I hadn't thought about what would make sense for `updates` - will change the method name here!